From 28fe7dd86ad6090baf3f9fbd76155c1c3aff8f99 Mon Sep 17 00:00:00 2001
From: Miklos Vajna <vmiklos@frugalware.org>
Date: Sat, 5 Dec 2009 04:57:10 +0100
Subject: [PATCH] Revert "drm/i915: use ACPI LID status for LVDS ->detect hook"

This reverts commit b42d4c5c6a872815d711e5d51a600f5122c38eee.

Conflicts:

	drivers/gpu/drm/i915/intel_lvds.c
---
 drivers/gpu/drm/i915/intel_lvds.c |    7 +------
 1 files changed, 1 insertions(+), 6 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_lvds.c b/drivers/gpu/drm/i915/intel_lvds.c
index 05598ae..2c4da8d 100644
--- a/drivers/gpu/drm/i915/intel_lvds.c
+++ b/drivers/gpu/drm/i915/intel_lvds.c
@@ -620,12 +620,7 @@ static const struct dmi_system_id bad_lid_status[] = {
  */
 static enum drm_connector_status intel_lvds_detect(struct drm_connector *connector)
 {
-	enum drm_connector_status status = connector_status_connected;
-
-	if (!acpi_lid_open() && !dmi_check_system(bad_lid_status))
-		status = connector_status_disconnected;
-
-	return status;
+	return connector_status_connected;
 }
 
 /**
-- 
1.6.5.3

